internal/worker/server.go

```diff
@@ -442,8 +442,12 @@ func (s *Server) requestJob(ctx context.Context, arch string, jobTypes []string,
 	if err != nil {
 		logrus.Errorf("error retrieving job status: %v", err)
 		return
-	} else {
-		prometheus.DequeueJobMetrics(status.Queued, status.Started, jobType)
+	}
+
+	if jobType == "osbuild:"+arch {
+		jobType = "osbuild"
```

Comment:
This makes sense, we should probably do the same in other places for consistency otherwise we get this:

![image](https://user-images.githubusercontent.com/20438192/158565982-9168bcfb-e030-48c0-9e54-966264483f48.png)

Reply:
oops. will do

Reply:
Done!